README.md

```diff
@@ -2,10 +2,14 @@
 
 # MomentOpt.jl: Moment Optimization Library for Julia
 
-Linux/MacOS: [![Build Status](https://travis-ci.org/floswald/MomentOpt.jl.svg?branch=master)](https://travis-ci.org/floswald/MomentOpt.jl)
+Linux/MacOS: [![Build Status](https://travis-ci.com/JulienPascal/MomentOpt.jl.svg?token=qvUh77uHnTEg8Fi4ytAR&branch=master)](https://travis-ci.com/JulienPascal/MomentOpt.jl)
```

Comment:
All of those badges refer that your accounts. You nchecked that the tests pass on my Travis accoInt?

Reply:
Yes you are right. They should point to your account. I'll change this.